setup.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+"""
+Package setup for krux-marathon-api
+"""
+######################
+# Standard Libraries #
+######################
+from __future__ import absolute_import
+from setuptools import setup, find_packages
+
+VERSION = '0.0.1'
```

Comment:
This is an old convention I am about to change. Please put `VERSION` in `__init__.py`. Refer to [this PR](https://github.com/krux/python-krux-cloud-health/pull/13).